src/ui/views/menu.coffee

```diff
@@ -1,74 +1,168 @@
 remote = require('electron').remote
 Menu = remote.Menu
 
-templateOsx = (viewstate) -> [{
-    label: 'YakYak'
-    submenu: [
-        { label: 'About YakYak', selector: 'orderFrontStandardAboutPanel:' }
-        { type: 'separator' }
-        # { label: 'Preferences...', accelerator: 'Command+,', click: => delegate.openConfig() }
-        { type: 'separator' }
-        { label: 'Hide YakYak', accelerator: 'Command+H', selector: 'hide:' }
-        { label: 'Hide Others', accelerator: 'Command+Shift+H', selector: 'hideOtherApplications:' }
-        { label: 'Show All', selector: 'unhideAllApplications:' }
-        { type: 'separator' }
-        { label: 'Open Inspector', accelerator: 'Command+Alt+I', click: -> action 'devtools' }
+acceleratorMap = {
+    hideyakyak:           {others: 'Control+H', darwin:'Command+H'}
```

Comment:
I did it to make sure the mapping was correct and I didn't do something stupid copying and pasting.

I believe in conventions, so for me the argument that throughout the code it won't be like this wins me over :)